src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
     path: row.path,
     organizationId: row.org_id,
-    usersCount: parseInt(row.users_count)
+    usersCount: parseInt(row.users_count),
```

Comment:
Maybe this is only a remnant of the past and it has been changed, but IIRC parseInt doesn't default to 10, but depending on the input it can be assumed to be 8, 10 or 16: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

We will probably always receive inputs that will default to decimals here, but I think it is a good practice to always specify the radix